homeassistant/components/recorder/models.py

```diff
@@ -638,7 +641,10 @@ def as_dict(self) -> dict[str, Any]:  # type: ignore[override]
             last_changed_isoformat = process_timestamp_to_utc_isoformat(
                 self._row.last_changed
             )
-            if self._row.last_changed == self._row.last_updated:
+            if (
+                self._row.last_changed == self._row.last_updated
+                or self._row.last_updated is None
```

Comment:
Since this happens frequently, maybe reverse this if-statement as checking for None is faster.
```suggestion
                self._row.last_updated is None
                or self._row.last_changed == self._row.last_updated
```

Reply:
Thanks